core/dbt/events/types.py

```diff
@@ -395,7 +395,7 @@ def code(self) -> str:
     def message(self) -> str:
         description = (
             f"The `{self.deprecated_path}` config has been renamed to `{self.exp_path}`. "
-            "Please update your `dbt_project.yml` configuration to reflect this change."
+            "Please see https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more information."
```

Comment:
Would this link: https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax be more informative?

cc @graciegoheen

Reply:
Yes, but then we should update that page to also have an example of `dbt_project.yml` cc: @matthewshaver

Reply:
The places folks will need to update this is in their properties yml files:
```
models:
  - name: orders
    columns:
      - name: order_id
        data_tests:
          - unique
          - not_null
```
and `dbt_project.yml`:
```
data_tests:
  +store_failures: true
```

Reply:
I confirmed that this error is raised only once, regardless of how many `tests` blocks there are (across both `properties.yml` and `dbt_project.yml` files), so I think it's fine to link to the docs page instead of including a file path (I'm not sure that we can get this, either)?

Reply:
https://github.com/dbt-labs/docs.getdbt.com/pull/5168